docs/contribute/style-guide.md

```diff
@@ -0,0 +1,182 @@
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>This command starts a proxy.</td><td>This command will start a proxy.</td></tr>
+</table>
+
+Exception: Use future or past tense if it is required to convey the correct
+meaning.
+
+### Use active voice
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>You can explore the API using a browser.</td><td>The API can be explored using a browser.</td></tr>
+  <tr><td>The YAML file specifies the replica count.</td><td>The replica count is specified in the YAML file.</td></tr>
+</table>
+
+Exception: Use passive voice if active voice leads to an awkward construction.
+
+### Use simple and direct language
+
+Use simple and direct language. Avoid using unnecessary phrases, such as saying "please."
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>To create a ReplicaSet, ...</td><td>In order to create a ReplicaSet, ...</td></tr>
+  <tr><td>See the configuration file.</td><td>Please see the configuration file.</td></tr>
+  <tr><td>View the Pods.</td><td>With this next command, we'll view the Pods.</td></tr>
+
+</table>
+
+### Address the reader as "you"
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>You can create a Deployment by ...</td><td>We'll create a Deployment by ...</td></tr>
+    <tr><td>In the preceding output, you can see...</td><td>In the preceding output, we can see ...</td></tr>
+</table>
+
+
+## Content anti-patterns
+
+### Avoid using "we"
+
+Using "we" in a sentence can get very confusing, because the reader often doesn't know if they're part of the "we" you're describing. In most instances, you want to avoid using "we". 
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Version 1.4 includes ...</td><td>In version 1.4, we have added ...</td></tr>
+  <tr><td>Kubernetes provides a new feature for ...</td><td>We provide a new feature ...</td></tr>
+  <tr><td>In this page, we are going to learn about pods.</td><td>This page will teach you how to use pods.</td></tr>
+</table>
+
+### Avoid jargon and idioms
+
+Some readers speak English as a second language. Avoid jargon and idioms to help make their understanding easier.
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Internally, ...</td><td>Under the hood, ...</td></tr>
+    <tr><td>Create a new cluster.</td><td>Turn up a new cluster.</td></tr>
+</table>
+
+### Avoid statements about the future
+
+Avoid making promises or giving hints about the future. If you need to talk about
+an alpha feature, put the text under a heading that identifies it as alpha
+information.
+
+{% endcapture %}
+
+
+{% capture whatsnext %}
+* Learn about [writing a new topic](/docs/contribute/write-new-topic/).
+* Learn about [using page templates](/docs/contribute/page-templates/).
+* Learn about [staging your changes](/docs/contribute/stage-documentation-changes/)
+* Learn about [creating a pull request](/docs/contribute/create-pull-request/).
+{% endcapture %}
+
+{% include templates/concept.md %}
```

Comment:
I might also suggest as an anti-pattern:

"Avoid referring to tasks as "simple", "easy", "complex", etc. Such modifiers are unnecessary and often subjective."